docs/api/curl/UmbrellaConectorDataExchange.md

```diff
@@ -0,0 +1,223 @@
+## Data Exchange Tutorial
+In order to start exchanging data, it will be necessary to install the subset with the necessary components for data exchange:
+```
+helm install umbrella -f ./charts/umbrella/values-adopter-data-exchange.yaml ./charts/umbrella/ --namespace umbrella
```

Comment:
Please reference to this part of the readme https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/README.md#option-2 or https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/README.md#option-2-1 respectively to not maintain helm install commands in multiple places